url-slug.cabal

```
@@ -24,7 +24,7 @@ extra-source-files:
 library
   build-depends:
     , aeson
-    , base                >=4.13.0.0 && <=4.18.0.0
+    , base                >=4.13.0.0 && <4.20
```

Let's just relax it permanently.

```suggestion
    , base                >=4.13.0.0 && <5
```